api/url_mapping.go

```diff
@@ -60,15 +60,15 @@ func MapUrl(app *fiber.App, handlers ...fiber.Handler) {
 	coreGroup := v1.Group("core")
 	//secret group
 	secrets := coreGroup.Group("secrets")
-	secrets.Post("/", secret.Create)
+	secrets.Post("/", secret.Create, middleware.IsDuplicated)
 	secrets.Head("/", secret.Count)
 	secrets.Get("/", secret.List)
 	secrets.Get("/:name", secret.ValidateSecretExist, secret.Get)
 	secrets.Put("/:name", secret.ValidateSecretExist, secret.Update)
 	secrets.Delete("/:name", secret.ValidateSecretExist, secret.Delete)
 	//storage class group
 	storageClasses := coreGroup.Group("storageclasses")
-	storageClasses.Post("/", storage_class.Create)
+	storageClasses.Post("/", storage_class.Create, middleware.IsDuplicated)
```

Comment:
we're not creating storage classes, you can remove this route altogether.
checking for conflict with stateful set is not needed.